android-components/components/browser/menu/src/main/res/layout/mozac_browser_menu_item_image_text_checkbox_button.xml

```diff
@@ -28,6 +29,7 @@
         android:background="@android:color/transparent"
         android:gravity="center_vertical"
         android:importantForAccessibility="no"
+        android:paddingEnd="8dp"
```

Comment:
Because this is an Android Components, we don't want to hardcode these padding values like this. The styles are meant to be configurable by the consumer. https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/browser/menu/README.md#18

Reply:
It looks like doing it through the @styles would require some additional work, to decouple checkbox button from the regular menu button. Those use the same styling for container and for the text view. At the same time, checkbox style doesn't have spacing in its style...

Since the change is needed on the fenix side, how about we extend BrowserMenuImageTextCheckboxButton and provide a new xml in`getLayoutResource`?